services/vso/vso.tester.js

```diff
@@ -42,37 +45,13 @@ t.create('build status with connection error')
 // Releases
 
 t.create('release status is succeeded')
-  .get('/release/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/77/16.json')
```

Comment:
Is there a good reason for removing the mocked tests here instead of switching `nock` to mock the request to https://vsrm.dev.azure.com instead e.g:

```js
t.create('release status is succeeded')
  .get('/release/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/77/16.json')
  .intercept(nock =>
    nock('https://vsrm.dev.azure.com')
      .get(
        '/devdiv/_apis/public/Release/badge/0bdbc590-a062-4c3f-b0f6-9383f67865ee/77/16'
      )
      .reply(200, '<svg><g><text>succeeded</text></g></svg>')
  )
  .expectJSON({ name: 'deployment', value: 'passing' })
```

and so on

(I've not tested that code, but hopefully you get the idea)

Reply:
Actually, when I introduced these "release" tests at first, someone asked me why I mocked them instead of doing the same thing as for "build" tests. So what I did here is to build an actual Azure DevOps project (see line 14: https://dev.azure.com/totodem/Shields.io) with a build pipeline and a release one in order to run integration tests.

It seems that's how tests are usually done here, doesn't it?

Reply:
Right, yep. Hadn't clocked there was no 'live' test for `/release`. Seems reasonable :)